guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -29,6 +29,11 @@ endif::[]
 ifdef::client-content-apt[]
 * On Debian: Check the `/etc/apt/sources.list` file and ensure that the appropriate repositories have been enabled.
 endif::[]
+ifeval::["{context}" == "load-balancing"]
```

Comment:
I suggest `ifdef::load-balancing[]` for guide-specific conditions.
However, I see it's a pre-existing problem for other occurences, so consider this non-blocking.

Reply:
That would be nice to have, I agree. If you have time.

Reply:
One more thing - I'm not sure since which version the attribute is present. So it's a good idea to check that when cherry-picking.